px-datetime-picker.html

```diff
@@ -53,36 +59,64 @@
     </px-datetime-field>
     <iron-dropdown
       id="dropdown"
-      opened="{{_opened}}"
-      vertical-offset="20"
-      dynamic-align
-      no-overlap>
+      class="dt-iron-dropdown"
+      opened="{{opened}}"
+      vertical-align="top"
+      horizontal-align="left"
+      position-target="[[_getPositionTarget(fillContainer, fitIntoElement)]]"
```

Comment:
There's a separate bug/behavior in the current component that I want to see if we can address with positioning (probably my fault from when I first switched to iron-dropdown), let's discuss whether it makes sense to handle now or backlog.